components/financial_assistance/spec/domain/financial_assistance/operations/applications/transformers/application_to/cv3_application_spec.rb

```diff
@@ -18,7 +18,7 @@
                                   gender: person.gender,
                                   ssn: person.ssn,
                                   application: application,
-                                  ethnicity: [],
+                                  ethnicity: nil,
```

Comment:
i changed this to nil to make sure that the transform changes it to an empty array fyi